components/cta_bar/index.styl

```diff
@@ -1,14 +1,69 @@
 @import '../stylus_lib'
 
+.cta-bar.editorial-signup
+  padding 0
+  background none transparent
+  background linear-gradient(to bottom, rgba(0,0,0,0), rgba(0,0,0,.25))
+  display flex
+  align-items flex-end
+  min-height 35vh
+  .vam-outer
+    background-color white
+    color black
+    padding 10px 20px
+    margin 10px
+    width 100%
+    border 1px solid #E5E5E5
+    .cta-bar-header-editorial
+      margin 10px 0 15px 0
+      > h2
+        garamond-size('s-headline')
+        text-align left
+        line-height 1.25
+    .cta-bar-defer a
+      font-size 22px
+      color #ccc
+      text-decoration none
+    .cta-bar-form-editorial
+      position relative
+      .bordered-input
+        border-width 1px
+        font-size 18px
```

Comment:
for font sizes, check out this style guide: https://www.artsy.net/style-guide/typography